src/darwin/Framework/CHIP/templates/templates.json

```diff
@@ -35,6 +35,11 @@
             "path": "clusters-tests.zapt",
             "name": "Cluster Tests",
             "output": "src/darwin/Framework/CHIPTests/CHIPClustersTests.m"
+        },
+        {
+            "path": "test-javafile.zapt",
+            "name": "test java file",
+            "output": "src/darwin/Framework/CHIP/gen/Test.java"
         }
```

Comment:
Can we move that to `src/controller/java/templates/templates.json` ? It would make more sense that having this living into darwin specific templates.

Reply:
`Test.java` was just for verifying that the ZAP templates action could pass. I've removed it and its template now.